.github/workflows/test_policy_sql.yml

```diff
@@ -0,0 +1,83 @@
+name: SQL Policy Validation Test
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+env: 
+  CGO_ENABLED: 0
+  CQ_NO_TELEMETRY: 1
+
+jobs:
+  SQLPolicyTest:
+    strategy:
+      matrix:
+        dbversion: [ "postgres:10" ]
+        go: [ "1.17" ]
+        platform: [ ubuntu-latest ] # can not run in macOS and widnowsOS
+    runs-on: ${{ matrix.platform }}
+    services:
+      postgres:
+        image: ${{ matrix.dbversion }}
+        env:
+          POSTGRES_PASSWORD: pass
+          POSTGRES_USER: postgres
+          POSTGRES_DB: postgres
+        ports:
+          - 5432:5432
+        # Set health checks to wait until postgres has started
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+    steps:
+      - name: Check out code into the policy directory
+        uses: actions/checkout@v3
+
+      - name: Download Cloudquery
+        run: |
+          curl -L https://github.com/cloudquery/cloudquery/releases/latest/download/cloudquery_Linux_x86_64 -o cloudquery
+          chmod a+x cloudquery
```

Comment:
I think we should be using the Github Action that we have on the marketplace

Reply:
This file is mostly a copy of `test_policy.yml` (synced from [here](https://github.com/cloudquery/.github/blob/main/workflows/providers/test_policy.yml) which runs the current `cq policy` command testing) but instead we do the `psql` thing after creating the tables with `cq sync`.